src/locales/index.ts

```diff
@@ -11,7 +10,6 @@ import sk from './sk';
  * in romcal.
  */
 const locales = {
-  'en-CA': enCA,
```

Comment:
You are removing the only one localisation file for a specific country/region.
In the top of this file, we might add a basic example (in a comment block) if someday we have to reintroduce one again, especially about the naming and the common conventions for this type of data (xx-XX and xxXX).

Reply:
Yeah, I am removing it as currently, it is not useful at all. All regional locales should contain only differences (in comparison with the ‘base’ locale) from the l10n/linguistic point of view; it should not create new keys.

> In the top of this file, we might add a basic example (in a comment block) if someday we have to reintroduce one again, especially about the naming and the common conventions for this type of data (xx-XX and xxXX).

Well, I see want you mean, however, why would be add a placeholder file like this into the romcal codebase? I think that better place for this is the documentation.

Reply:
It's a very specific things and it's often useful to have this type of instructions directly in the codebase. So we don't have to search how to do it when we have to, especially when it's a code specific instruction. And also because we generally don't read documentations, until we are block since a moment :)

So I think it's a good thing to add just a very basic example here, and it will improve future contributions experience.

Reply:
From my point of view, it is not the best place. I think that if we once have a localisation tool _and_ a website/web-app that would make it possible to create a new locale, that note would be unnecessary.

If you, however, really need that example in the codebase, what about `src/locales/index.ts`? We could prepend the example their. It might be even sufficient to note that regional locales (like `en-CA.ts`) should have `xx-XX.ts` format.

> And also because we generally don't read documentations, until we are block since a moment :)

This is not the case with me. I always read the docs, man pages and any help I could get. :smiley: I really miss an ultimate documentation (be it user and dev documentation) of romcal.

> So I think it's a good thing to add just a very basic example here, and it will improve future contributions experience.

If you really insist, send me a patch or a text example.

Reply:
I was speaking about adding an example just above in `src/locales/index.ts`. But after looking at the entire file, I just saw that we still have the 'pt-BR: ptBR` (I forgot it had been added recently). So, it's all right 👍